packages/blade/dependencies-external-plugin.mjs

```diff
@@ -0,0 +1,68 @@
+// Modified from: https://github.com/pmowrer/rollup-plugin-peer-deps-external/blob/master/src/index.js#L3
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { either } from 'ramda';
+const isFunction = (fn) => typeof fn === 'function';
+
```

Comment:
what did we change in this plugin from the original source?

Reply:
Nothing, just kept it as is.

Reply:
arre then why did we moved it here?

Reply:
Because i cannot import `.ts` file in this `.mjs` file without first transpiling it.